internal/states/encryption/flow/mockup.go

```diff
@@ -0,0 +1,50 @@
+package flow
+
+import (
+	"github.com/hashicorp/go-hclog"
+	"github.com/opentofu/opentofu/internal/logging"
+	"github.com/opentofu/opentofu/internal/states/encryption/encryptionconfig"
+)
+
+// MockUpLoggingFlow will be removed when we replace it with the real implementation.
+type MockUpLoggingFlow struct {
+	configKey string
+	logger    hclog.Logger
+}
+
+func New(configKey string) Flow {
```

Comment:
To make sure nobody mistakes this for an actual production flow:

```suggestion
func NewMock(configKey string) Flow {
```